frontend/src/app/App.tsx

```diff
@@ -82,7 +82,7 @@ const App: React.FC = () => {
 
     // Default error handling for other cases
     return (
-      <Page>
+      <Page className="pf-m-no-sidebar">
```

Comment:
Is there really no way to avoid adding a class name? This seems to be breaking the idea of React PF

Reply:
Great question @andrewballantyne - in other use cases, we could achieve this with `sidebar={null}`. However, when the server is shut down, the React component cannot re-render, so in this scenario we do need to apply the classname.

Reply:
Why can't the react cycle re-render? Losing a server connection shouldn't mean we lose React render cycles...